_studio/hevce_hw/h265/include/mfx_h265_encode_plugin_hw.h

```diff
@@ -34,7 +34,7 @@ class Plugin : public MFXEncoderPlugin
 public:
     static MFXEncoderPlugin* Create()
     {
-        return new Plugin(false);
+        return GetInstance();
```

Comment:
I guess FEI also suffers from such issue?

Reply:
+ Old design also had such behavior of function create. Why now there is a memory leak?

Reply:
FEI shouldn't be affected by this issue.
Early (actual for FEI) this object delete by  mfx_sessing during closing, but  
after moving HW HEVCe to library  USER_Register() for HW HEVCe guid do nothing, so mfx_session don't know about plugin.
So nobody deletes object Plugin(), issue reproduced each time when HE HEVCe loaded as plugin (used guid)